ansible_bender/schema.py

```diff
@@ -223,7 +223,8 @@
             "examples": [
                 "/usr/bin/python4"
             ]
-        }
+        },
+    "additionalProperties": False
```

Comment:
this is the schema for the db file, users are not meant to change this one, hence we don't need to validate the file so striuctly. Please set it in the other schema, the defined below - for the vars section. Thank you!

Reply:
@TomasTomecek I was actually confused, thank you for helping me out.
I found that keys such as`verbose_layer_names` were not there in that schema.